content/en/apps/guides/performance/purging.md

```diff
@@ -106,6 +106,8 @@ This function takes four parameters:
 - `contact`, the contact document of a patient or other contact who has reports about them.
 - `reports`, an array of all reports for that subject that are present on the server.
 - `messages`, an array of sms messages that the contact has sent or received
+- `chtScriptApi` the CHT API that provides CHT-Core Framwework's functions to toher parts of the app. More info on thea API can be found [here](https://docs.communityhealthtoolkit.org/apps/reference/_partial_cht_api/)
+- `persmissionSettings` string or array of permissions that the user has. More info on permissions can be found [here](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/user-permissions/)
```

Comment:
Same here about the link notation, example `[tasks]({{<ref "apps/reference/tasks" >}})`

Reply:
Are any of these optional?